src/auto-evo/RunResults.cs

```diff
@@ -599,6 +599,14 @@ public void PrintSummary(PatchMap? previousPopulations = null)
             LocalizedStringBuilder PatchString(Patch patch)
             {
                 var builder2 = new LocalizedStringBuilder(80);
+
+                if (!patch.Explored)
```

Comment:
I think this method needs to have a parameter to skip Fog of war, so that the report to be shown in the auto-evo exploration tool works well.